Arduino-Bling/RobotBling/RobotBling.ino

```diff
@@ -81,19 +81,20 @@ const rgbColor rgbColor_MAGENTA_DIM(0x330033);
    Animation index
     - List of animation used by Spartronics -- MUST match LED subsystem BlingStates
     - loop() matches the animation to the command entered on the serial port
+    - Note: java naming convention used to keep Dana happy :)
 */
 enum
```

Comment:
in c, i believe you can say:

enum Bling
{
} currentCommand;

or enum Bling {};
then Bling currentCommand;  // if this doesn't work we'd use typedef enum Bling Bling;

then your current command could be of this type.  Now when you send it across the wire you can explictly cast to uint8_t;